workbench-core/base/src/utilities/queryParamsFiltersUtils.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import { marshall } from '@aws-sdk/util-dynamodb';
+import Boom from '@hapi/boom';
+import { QueryParams } from '../aws/helpers/dynamoDB/dynamoDBService';
```

Comment:
Also I think `queryParams.ts` should go into your interfaces folder. I had it in constants but interfaces makes more sense. Can you move when you merge `develop` into this branch?

Reply:
added to interfaces